definitions/apm-application/relationships.yml

```diff
@@ -0,0 +1,26 @@
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  - instrumentation.provider: newrelic
```

Comment:
super mega nit: newRelic is the default one

Reply:
updated!